tests/test_api.py

```diff
@@ -101,6 +102,15 @@ def test_request_post(self):
             self.assertEqual(r.request.body,
                              '{"payload": "this is my payload."}')
 
+    def test_connection_ssl(self):
+        with client.test_mode as t:
+            t.register_json('/ping/', {'status': 'ok'})
+            https_adapter = client.adapters['https://']
+            with mock.patch.object(FauxAdapter, 'send', wraps=https_adapter.send) as mock_send:
+                client.get('/ping/')
+                mock_send.assert_called_once()
+                self.assertTrue(mock_send.call_args[1]['verify'])
```

Comment:
```python
                mock_send.assert_called_once_with(
                    mock.ANY, cert=None, proxies=mock.ANY, stream=mock.ANY,
                    timeout=mock.ANY, verify=True
                )
```

It looks like this would work with the python versions failing in Travis.